grails-app/migrations/views/stock-movement-list-item.sql

```diff
@@ -51,29 +51,29 @@ CREATE OR REPLACE VIEW stock_movement_list_item AS
         s.id AS shipment_id,
         s.current_status AS shipment_status,
         CASE
-            WHEN o.status IS NULL THEN 'PENDING'
-            WHEN o.status = 'PENDING' THEN 'PENDING'
-            WHEN o.status = 'PLACED' THEN 'REQUESTED'
+            WHEN o.status IS NULL THEN 'CREATED'
+            WHEN o.status = 'PENDING' THEN 'CREATED'
+            WHEN o.status = 'PLACED' THEN 'CREATED'
             WHEN o.status = 'APPROVED' THEN 'PICKING'
             WHEN o.status = 'CANCELED' THEN 'CANCELED'
             # These probably won't be used but we want to cover them just in case
-            WHEN o.status = 'PARTIALLY_RECEIVED' THEN 'DISPATCHED'
-            WHEN o.status = 'RECEIVED' THEN 'DISPATCHED'
-            WHEN o.status = 'COMPLETED' THEN 'DISPATCHED'
+            WHEN o.status = 'PARTIALLY_RECEIVED' THEN 'ISSUED'
+            WHEN o.status = 'RECEIVED' THEN 'ISSUED'
+            WHEN o.status = 'COMPLETED' THEN 'ISSUED'
             WHEN o.status = 'REJECTED' THEN 'CANCELED'
             ELSE 'PENDING'
         END AS status_code,
         NULL AS requisition_id,
         CASE
```

Comment:
Same comment as above

Reply:
Holy shit. This makes me very nervous and is the antithesis of what I wanted to do with these tickets. This is "possible long-term" refactor, not "just get it working" refactor. 

<img src="https://media1.giphy.com/media/ZOStzpF9H5syI/giphy.gif"/>

Reply:
As for this change I’m pretty confident, since this part of the code is responsible only for returns.